imod_coupler/drivers/dfm_metamod/config.py

```diff
@@ -127,6 +128,12 @@ def validate_dfm_model_is_mdu(cls, dfm_model: str) -> str:
             raise ValueError("the dflow fm model name should end in mdu")
         return dfm_model
 
+    @validator("output_config_file")
+    def validate_toml_file(cls, filename: FilePath) -> FilePath:
```

Comment:
optionally, you can replace matching extensions like this:
if os.path.splitext(filename.name)[1].lower() != "toml":

Reply:
done